sensor/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,116 @@
+# tag_based_pnp_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/tag_based_pnp_calibrator.md)
+
+## Purpose
+
+The package `tag_based_pnp_calibrator` allows extrinsic calibration among the camera sensor and lidar sensor used in autonomous driving and robotics.
+
+## Inner-workings / Algorithms
+
+The `tag_based_pnp_calibrator` utilizes the PnP algorithm to calculate the transformation between the camera and lidar. To run this package, you also need to operate the `apriltag_ros` package and the `lidartag` package to calculate the transformation.
```

Comment:
The verb does not operate here. We do not operate a package/node (in the way we operate a machine). We use, require, or execute

Reply:
`apriltag_ros` and the `lidartag` packages

Reply:
fix in https://github.com/tier4/CalibrationTools/pull/164/commits/32dd61431d3beb3d84ecb085cd246042de8efb09